composer.json

```diff
@@ -24,6 +24,9 @@
 		}
 	},
 	"scripts": {
-		"backport": "bin/backport-wp-commit.sh"
+		"backport": "bin/backport-wp-commit.sh",
+		"phpcs": "vendor/bin/phpcs -n",
+		"phpcs-tests": "vendor/bin/phpcs -n",
```

Comment:
These seem to do the same thing. Is `phpcs` an alias of `phpcs-tests`?

Reply:
Absolutely right - I missed adding the `tests` folder name - updated PR to follow